__tests__/app.test.js

```diff
@@ -0,0 +1,76 @@
+const db = require('../db/connection')
+const app = require('../app')
+const request = require('supertest')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+const endpoints = require('../endpoints.json')
+
+beforeEach(()=>{
+    return seed(testData)
+})
+afterAll(()=>{
+    return db.end()
+})
+describe('GET /*', ()=>{
+    test('should return a 404 status when an attempt to access a non existent endpoint is made', ()=>{
+        return request(app)
+        .get('/non-existing')
+        .expect(404)
+    })
```

Comment:
Don't forget to check the msg here too - this should be your message rather than the built in node response.

Reply:
Noted